diagnostic.md

```diff
@@ -73,14 +88,20 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl -include --request POST http://www.example.com/donuts5
 ```
 
 Write an AJAX request to create a single donut on the server using JSON. Please
 do not use `data = getFormFields(form)` instead write out the data object.
 
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function(id, data){
+  return $.ajax({
+    url: http://www.example.com + '/donuts/' + id,
+    method: 'POST',
+    data
```

Comment:
This will work but we want to see the object representing the French Cruller donut being passed in as well.